proposal/README.md

```diff
@@ -1 +1,94 @@
-proposal/requirement
+SUSE SolidDriver Checks
+=======================
+
+The purpose of the SUSE SolidDriver Checks is to ensure best end user
+experience by verifying that third party delivered drivers are built and
+deployed in a SUSE SolidDriver compliant] manner.
+
+General Features
+----------------
+*NOTE: Not all features are implemented at this time*
+
+* This tool shall be usable by SUSE, SUSE Partners, and SUSE customers.
+* The tool shall be able check running and installed modules on local or
+  remote systems
+* The tool shall be able check kernel module packages (KMPs) that have not
+  been installed
+* The tool shall create reports in the following formats:
+    - HTML
+    - Excel spreadsheet
+    - PDF
+    - JSON
+    - Terminal
+* Remote checks executed via ssh preferably using ssh keys for
+  authentication
+
+
+Checks
+------
+*NOTE: Not all checks are implemented at this time*
+
+* Critical checks failures shall give a high level warning (RED)
+* Important checks failures shall give a low level warning (AMBER)
+
+
+### Kernel Modules
+
+Critical checks
+
+* Kernel modules have `supported` flag properly set
+    - `supported` flag appears only once
```

Comment:
I don't check how many times the ```supported``` shown in the output. Is there any reason this appears multiple times?

Reply:
It can happen. I have seen it set to 'external' and 'yes' in the same module.